messaging/Controllers/BundlesController.cs

```diff
@@ -77,15 +79,29 @@ public async Task<ActionResult<Bundle>> GetOutgoingMessageItems(string jurisdict
                 return BadRequest("Pagination does not support specifying a page without a _since parameter");
             }
 
-            try
+            RouteValueDictionary searchParamValues = new()
             {
-                // Limit results to the jurisdiction's messages; note this just builds the query but doesn't execute until the result set is enumerated
-                IQueryable<OutgoingMessageItem> outgoingMessagesQuery = _context.OutgoingMessageItems.Where(message => (message.JurisdictionId == jurisdictionId));
+                { "jurisdictionId", jurisdictionId },
+                { "_count", _count }
+            };
+            if (certificateNumber != null) {

```

Comment:
This kept tripping me up while testing and I'm sure jurisdictions will do the same, pad certificateNumber with leading zeros so it is 6 characters long so even if the user provide "80" as the param it still works